spec/outline-spec.js

```diff
@@ -19,7 +19,12 @@ describe("Outline", () => {
     jasmine.attachToDOM(workspaceElement);
 
     waitsForPromise(() => atom.workspace.open(mockFilePath));
-    waitsForPromise(() => atom.packages.activatePackage("atom-ide-outline"));
+    // activation
+    waitsForPromise(() => {
+      atom.packages.triggerActivationHook("core:loaded-shell-environment");
+      atom.packages.activatePackage("atom-ide-outline");
+    });
+    atom.packages.triggerDeferredActivationHooks();
```

Comment:
`waitsForPromise` fails in this case (98cd86e), because it doesn't receive a promise. However, the solution is solving the right problem, package activation is being deferred and never triggered. 

```suggestion
    // Package activation will be deferred to the configured, activation hook, which is then triggered
    waitsForPromise(() => atom.packages.activatePackage("atom-ide-outline"));
    atom.packages.triggerActivationHook("core:loaded-shell-environment");
    atom.packages.triggerDeferredActivationHooks();
```